bin/greet

```diff
@@ -3,5 +3,5 @@ require_relative "../lib/greeting.rb"
 
 puts "Hi! I'm HAL, what's your name?"
 
-name = gets.chomp
+name = gets.strip
```

Comment:
I don't think this is right. chomp is the right one to use for "remove the trailing `\n`). Do we think that there will be `"    (input)   \n"`? That might explain why to use `strip`, but `chomp` seems right here.

Reply:
@sgharms If you have whitespace before or after your name, chomp doesn't work. I am adding some examples to illustrate why.